docs/accounts/how-to-access-mahti-large-partition.md

```diff
@@ -31,15 +31,12 @@ not affect results, typically few minutes for the shortest run time
 (largest node count) is fine.
 * Parameters affecting the scalability can, and are encouraged to be,
 changed. Note, also the [performance checklist](../computing/running/performance-checklist.md)
+* Scalability is considered good if the parallel efficiency is 85 %, minimum requirement is 75 % efficiency 
```

Comment:
Do we need to give such an exact number for what is good? Perhaps just state 75% is minimum but one should target higher. 

Also, should we state these in terms of speedup when doubling number of nodes, imho parallel efficiency is more difficult to understand and is not described here.

Reply:
Yes, I agree that the 85% should be dropped.

About the second point, speedup is already used as an explanation for the parallel efficiency (in the next line), so IMHO it's all good.